azure-release-pipelines.yml

```diff
@@ -18,6 +18,13 @@ parameters:
     displayName: PyTest Marker
     type: string
     default: 'not weekly'
+  - name: AGENT_POOL
+    displayName: Agent Pool
+    type: string
+    default: 'ScalingVMSet'
+    values:
+    - 'ScalingVMSet'
```

Comment:
Can you open an oncall issue to rename this to `linux-pool` or `linux-ci` to make it more consistent with the windows one?